openpype/hosts/maya/plugins/publish/validate_plugin_path_attributes.py

```diff
@@ -19,37 +21,50 @@ class ValidatePluginPathAttributes(pyblish.api.InstancePlugin):
     hosts = ['maya']
     families = ["workfile"]
     label = "Plug-in Path Attributes"
+    actions = [SelectInvalidAction]
 
-    def get_invalid(self, instance):
+    # Attributes are defined in project settings
+    attribute = []
+
+    @classmethod
+    def get_invalid(cls, instance):
         invalid = list()
 
-        # get the project setting
-        validate_path = (
-            instance.context.data["project_settings"]["maya"]["publish"]
-        )
-        file_attr = validate_path["ValidatePluginPathAttributes"]["attribute"]
+        file_attr = cls.attribute
         if not file_attr:
             return invalid
 
-        # get the nodes and file attributes
-        for node, attr in file_attr.items():
-            # check the related nodes
-            targets = cmds.ls(type=node)
+        # Consider only valid node types to avoid "Unknown object type" warning
+        all_node_types = set(cmds.allNodeTypes())
+        node_types = [key for key in file_attr.keys() if key in all_node_types]
 
-            for target in targets:
-                # get the filepath
-                file_attr = "{}.{}".format(target, attr)
-                filepath = cmds.getAttr(file_attr)
+        for node, node_type in pairwise(cmds.ls(type=node_types,
```

Comment:
Isn't `pairwise` Python 3.10 functionality?

Reply:
It's a function that's in maya lib api for quite some time - it's not using a python native implementation.
`pairwise` is imported at the top of the file.